src/rust/dbv2/migrations/2023-12-07-135237_metrics/up.sql

```diff
@@ -0,0 +1,89 @@
+-- Your SQL goes here
+CREATE VIEW aggregator.last_time_indexed AS
```

Comment:
So now it is `latest_event_time`, which is fine within the context of the order history pipeline

But for the context of the aggregator schema, `aggregator.latest_event_timestamp` is confusing, because this does *not* take into account the latest event timestamp from size changes, market orders, etc.

So how about `aggregator.order_history_latest_event_timestamp`?